apple/internal/experimental_mixed_language_library.bzl

```diff
@@ -188,16 +188,38 @@ generated if non-empty.
     ],
 )
 
-# TODO: Document the experimental_mixed_language_library macro
-def experimental_mixed_language_library(**kwargs):
+def experimental_mixed_language_library(
+        *,
+        name,
+        srcs,
+        deps = [],
+        module_name = None,
+        objc_copts = [],
+        swift_copts = [],
+        swiftc_inputs = [],
+        **kwargs):
     """Compiles and links Objective-C and Swift code into a static library.
```

Comment:
Can we add something here saying how's is discouraged and is mainly to aid people migrating? Not sure on the exact language, but something to explain why `experimental` is in the name.